### PR TITLE
Fix unsatisfiable reasoning query

### PR DESCRIPTION
**Pull request by flyingsilverfin**
## What is the goal of this PR?
Fix unsatisfiable concept inequality test

## What are the changes implemented in this PR?
* Fix test

### CODE REVIEW

File: graql/reasoner/concept-inequality.feature [hunk 1]
```diff
@@ -572,15 +569,12 @@ Feature: Concept Inequality Resolution
...
         $y has $unwantedValue;
-        not { $value is $unwantedValue; };
         $unwantedValue "Ocado";
-        $value isa! $type;
-        $unwantedValue isa! $type;
-        $type type base-attribute;
-        not { $type is $unwantedType; };
-        $unwantedType type string-attribute;
+        not { $value is $unwantedValue; };
       get $x, $value, $type;
```
Comment (jmsfltchr) [inline graql/reasoner/concept-inequality.feature:578]: `$type` is no longer present

File: graql/reasoner/concept-inequality.feature [hunk 2]
```diff
@@ -572,15 +569,12 @@ Feature: Concept Inequality Resolution
...
         $x has $value;
         $y has $unwantedValue;
-        not { $value is $unwantedValue; };
         $unwantedValue "Ocado";
-        $value isa! $type;
-        $unwantedValue isa! $type;
-        $type type base-attribute;
-        not { $type is $unwantedType; };
-        $unwantedType type string-attribute;
+        not { $value is $unwantedValue; };
```
Comment (jmsfltchr) [inline graql/reasoner/concept-inequality.feature:577]: According to the test name we're restricting type and attribute value separately, so it follows that this should be `$value != $unwantedValue;`

**Review by jmsfltchr — CHANGES_REQUESTED** [review]

File: graql/reasoner/concept-inequality.feature (hunk 2 above)
Comment (flyingsilverfin) [reply graql/reasoner/concept-inequality.feature]: addressed

File: graql/reasoner/concept-inequality.feature (hunk 1 above)
Comment (flyingsilverfin) [reply graql/reasoner/concept-inequality.feature]: interesting - this should probably be an issue in Graql

Comment (flyingsilverfin) [reply graql/reasoner/concept-inequality.feature]: addressed

**Review by jmsfltchr — APPROVED** [review]

**Pull request merged**